client/extensions/woocommerce/store-sidebar/index.js

```diff
@@ -118,3 +123,17 @@ export default class StoreSidebar extends Component {
 		);
 	}
 }
+
+function mapStateToProps( state ) {
+	const currentUser = getCurrentUser( state );
+	const selectedSiteId = getSelectedSiteId( state );
+	const isSingleSite = !! selectedSiteId || currentUser.site_count === 1;
+	const siteId = selectedSiteId || ( isSingleSite && getPrimarySiteId( state ) ) || null;
+	const site = getSite( state, siteId );
```

Comment:
Good idea - done (at least for the sidebar) in https://github.com/Automattic/wp-calypso/pull/14022/commits/2db9bb49f7dd460b832fee14dc3dadb9f7bd27fc